website/docs/secrets/managing-secrets-with-sops.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: Managing secrets with Sops
+hide_title: true
+---
+
+# Managing secrets with Sops
+
+Weave GitOps Enterprise now supports managing secrets using [SOPS](https://github.com/mozilla/sops), a tool that encrypts and decrypts secrets using various key management services.
```

Comment:
Let's remove this one altogether, It feels like an announcement, while it's just documentation. 
```suggestion

```